packages/postgres-datasource/tests/datasource.test.ts

```diff
@@ -128,7 +135,12 @@ describe('PostgresDataSource', () => {
       'SELECT * FROM dbos.transaction_completion WHERE workflow_id = $1',
       [workflowID],
     );
-    expect(txOutput.length).toBe(0);
+    expect(txOutput.length).toBe(1);
```

Comment:
I may have missed it, but do we have a test to run a transaction twice with the same workflow ID and see if it can correctly throw the same error?

Reply:
will add